kclvm/cmd/src/tests.rs

```diff
@@ -596,3 +596,22 @@ fn test_error_message_fuzz_unmatched() {
         }
     }
 }
+
+#[test]
+fn test_multi_import() {
```

Comment:
I still suggest placing this test in the resolver. The cmd test is a bit too much.

Reply:
fixed.